httphandler/httphandler.go

```diff
@@ -89,10 +90,13 @@ func NewHandler(conf config.Config) http.Handler {
 		Dial:                dialer.Dial,
 		DisableKeepAlives:   conf.KeepAlive,
 		MaxIdleConnsPerHost: int(conf.ConnLimit)}
-
+	backends := make([]*url.URL, len(conf.Backends))
+	for _, backend := range(conf.Backends) {
+		backends = append(backends, backend.URL)
```

Comment:
Why appending? You already have a slice with required length, so you can just assign values to indexes.